package-lock.json

```diff
@@ -771,7 +771,7 @@
                 "kang": "^1.3.0",
                 "lstream": "0.0.4",
                 "microtime": "2.1.6",
-                "oldcrc": "git://github.com/joyent/node-crc.git#fd0477833ac28be81ada23bf84879fbde9cd8750",
+                "oldcrc": "https://github.com/joyent/node-crc.git#fd0477833ac28be81ada23bf84879fbde9cd8750",
```

Comment:
I'm going to assume JSON parses out the #commit properly... if I do that from a CLI with git (doesn't matter which URL form), I get errors, but I expect this is not a problem here.